inductiva/fluids/scenarios/coastal_area/bathymetry.py

```diff
@@ -264,35 +265,40 @@ def plot(
               the minimum or maximum colors, respectively.
             path: Path to save the plot. If `None`, the plot is not saved, and
               the matplotlib `Axes` object is returned instead.
-            grid_size: Size of the uniform grid to which the bathymetry is
-              interpolated before being plotted.
-            max_distance: Threshold distance to filter out points on the
+            grid_resolution: Resolution, in meters, of the uniform grid to which
+              the bathymetry is interpolated before being plotted.
+            threshold_distance: Threshold distance to filter out points on the
               uniform grid that are far from points where the bathymetry is
               defined.
         """
 
-        # Create uniform grid for interpolation.
-        x_grid, y_grid = np.meshgrid(np.linspace(*self.x_range, grid_size),
-                                     np.linspace(*self.y_range, grid_size))
+        # Determine grid size based on ranges and resolution.
+        x_num = int((self.x_range[1] - self.x_range[0]) / grid_resolution)
+        y_num = int((self.y_range[1] - self.y_range[0]) / grid_resolution)
 
-        # Interpolate depths to uniform grid.
-        interpolator = scipy.interpolate.LinearNDInterpolator(
-            (self.x, self.y),
-            self.depths,
-        )
+        logging.info(
+            "Plotting the bathymetry on a uniform grid...\n"
+            "- grid resolution %f m \n"
+            "- grid size %d x %d", grid_resolution, x_num, y_num)
 
-        depths_grid = interpolator(x_grid, y_grid)
+        if x_num > 1000 or y_num > 1000:
```

Comment:
If it is 1000x10 is it still large?

Reply:
Not really. I'll update the check to `if x_num * y_num > 1000**2`.